test/opentok-test.js

```diff
@@ -983,15 +975,14 @@ describe('#generateToken', function () {
     expect(function () {
       this.opentok.generateToken(this.sessionId, { expireTime: oneHourAgo });
     }).to.throw(Error);
-
+*/
```

Comment:
Lets delete this test

Reply:
These were actually good tests, I forgot to uncomment them (the way these are structured I couldn't easily just run the one assertion). Corrected.